tensorflow/python/keras/engine/base_layer.py

```diff
@@ -2152,7 +2152,7 @@ def output(self):
     """Retrieves the output tensor(s) of a layer.
 
     Only applicable if the layer has exactly one output,
-    i.e. if it is connected to one incoming layer.
+    i.e. if it is connected to one outgoing layer.
```

Comment:
I am not sure this is right

Reply:
Sorry for inconvenience, I realised that but forgot to commit the same. The change is incorrect, please revert.

Reply:
You have to revert it by pushing a new commit.